pyaedt/application/Design.py

```diff
@@ -4020,32 +4021,61 @@ def set_temporary_directory(self, temp_dir_path):
         return True
 
     @pyaedt_function_handler()
+    def _manipulate_design_settings_inputs(self, k, v):
+        return v
+
+    @property
     def design_settings(self):
-        """Get design settings for the current AEDT app.
+        return DesignSettings(self, self._manipulate_design_settings_inputs)
```

Comment:
Descriptions really shouldn't contain code entities. If this is a property (given the decorator), it should be a noun-based description rather than having a description that starts with a verb (like a method or function). Can't the description be "Design settings for the AEDT app."